ballerina-tests/tests/records.bal

```diff
@@ -318,3 +318,14 @@ public type Reviews readonly & record {|
     }
     int stars;
 |};
+
+type BookRow record {
+    readonly int id;
+    string title;
+    int author;
+};
```

Comment:
Better to use closed records if possible.

Reply:
Fixed